.github/workflows/e2e-tests.yaml

```diff
@@ -0,0 +1,55 @@
+name: E2E tests
+
+on: [pull_request]
+
+jobs:
+  e2e-tools:
+    name: E2E - Tools
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    env:
+      TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+      TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+          show-progress: false
+
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 8
+
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: "pnpm"
+
+      - name: Install dependencies
+        run: pnpm install
+
+      - name: Install browsers
+        run: pnpm exec playwright install chromium
+        working-directory: packages/apps/tools
+
+      - name: Build Tools dApp
+        run: pnpm --filter @kadena/tools...  build
+        env:
+          FAUCET_PUBLIC_KEY: 782127638ab9cc8fa8598ff0120464ceef6f367ddcd616b47afffbdb175dcc5e
```

Comment:
We can potentially fetch these from github environment variables, but they are dummy values so it might be too much overhead.

Reply:
I'll add a comment. Unfortunately the _values_ need to be syntactically correct so I can't just put 'dummy-value' as a value.